README.md

```diff
@@ -209,6 +209,7 @@ Modules or frameworks for virtual machines that run in the Cosmos SDK
 * [informalsystems/hermes](https://github.com/informalsystems/hermes) - IBC Relayer in Rust.
 * [confio/ts-relayer](https://github.com/confio/ts-relayer) - IBC Relayer in TypeScript.
 * [local-interchain](https://github.com/Reecepbcups/local-interchain) - Quickly spin up a local IBC development environment on any operating system.
+* [IBC-escrow-auditor](https://github.com/Cordtus/escrow-checker) - Standalone tool to check and report on-chain token amounts against the IBC lockup account on counterparty chain.
```

Comment:
Hey! Why adding your fork instead of the original repo: https://github.com/strangelove-ventures/escrow-checker
Your changes are minimal, and I'd rather, even if archived, add the previous repo.